printer_mods/zruncho/V0_No_Drop_Nuts/README.md

```diff
@@ -1,10 +1,10 @@
-# V0 No-Drop Nuts
+# V0 No-Drop Nuts (LDO only)
```

Comment:
Change `LDO only` to `LDO frame only` or `LDO extrusions only` to avoid confusion, since LDO makes many products.